### ISSUE TITLE
 Cordova-plugin-locationservices - error ionic

### ISSUE THREAD

**Issue by artbase** [louisbl/cordova-plugin-locationservices#2]
When I do below commands:
- ionic start testApp blank -a "Test App" -i pl.foo.testapp
- ionic platform add android 
- ionic plugin add https://github.com/MobileChromeApps/google-play-services
- ionic plugin add https://github.com/louisbl/cordova-locationservices

I get error "Error during processing of action .. "

https://cdn.discourse.org/business/uploads/ionicframework/original/2X/a/ac99d71267ed5d044007f036acc70d544d70b461.png

What I do wrong? Early this case worked good.

**Comment by artbase** [comment]
I add additional info, maybe can help you

Cordova CLI: 4.3.0.
Ionic Version: 1.0.0-rc.0
Ionic CLI Version: 1.6.4
Ionic App Lib Version: 0.3.8
OS: Windows 8.1
Node Version v0.12.0

**Comment by louisbl** [comment]
I'm not sure if it's related but since 863ef761 you don't need to add the google-play-services plugin.
The [installation guide](https://github.com/louisbl/cordova-plugin-locationservices#installation) has been updated regarding this change.

Would you mind sharing your cordova android version as well?

``` bash
$ cordova platform version android
```

**Comment by artbase** [comment]
I have android 4.0.2

![3](https://cloud.githubusercontent.com/assets/3199834/9086868/7ed6c46e-3b87-11e5-9683-48595a81db78.png)

**Comment by louisbl** [comment]
So the only way I managed to reproduce your bug was with an incorrect installation of the android-sdk.
Without adding the plugin, are you able to build your project for android? Have you set your `ANDROID_HOME` var?
What version of Android are you targeting?

**Comment by artbase** [comment]
It turned out that I have two Android SDK Managers installed on my computer. I made the update in the "Extras" in the SDK that is not used by Ionic. After the update of SDK used by Ionic and after removing the plug-in which shares the Google Play Services, cordova-plugin-googleplayservices (recently it is not needed) the project began to build properly and works well now.

In addition I updated CLI Cordova to 5.1.1 version

Thanks your help
